chatgpt-shell-google.el

```diff
@@ -51,9 +51,19 @@ If you use Gemini through a proxy service, change the URL base."
   :safe #'stringp
   :group 'chatgpt-shell)
 
+(defcustom chatgpt-shell-google-thinking-budget-tokens -1
```

Comment:
nit: Is it possible to replace `-1` with `'dynamic`? Gets rid of magic number needing comment.